storage/market.go

```diff
@@ -59,9 +60,14 @@ func (s *Storage) SaveRates(rates blockatlas.Rates, pl ProviderList) {
 		}
 		err = s.AddHM(EntityRates, rate.Currency, &rate)
 		if err != nil {
+			errCounter++
 			logger.Error(err, "SaveRates", logger.Params{"rate": rate})
 			continue
 		}
+		if errCounter == len(rates) {
+			return errors.E("SaveRates all")
```

Comment:
maybe it's better to try to save one of these errors to put inside the return error context

Reply:
It is an awesome idea